src/platform/models/browser/components/ViewModeToggle.vue

```
@@ -0,0 +1,46 @@
+        )
+      "
+      :aria-label="$t('modelBrowser.viewList')"
+      @click="$emit('update:modelValue', 'list')"
+    >
+      <i class="icon-[lucide--list] size-3.5 sm:size-4" />
+    </button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { cn } from '@/utils/tailwindUtil'
```

ESLint

[error] Unable to resolve path to module '@/utils/tailwindUtil'.

(import-x/no-unresolved)